src/CoreBundle/private/library/classes/TCMSURLHistory.class.php

```diff
@@ -21,7 +21,17 @@ class TCMSURLHistory
      */
     public $aHistory = array();
 
-    public $index = 0;
+    /**
+     * @deprecated since 6.3.0 - use getHistoryIndex() instead
+     */
+    // public $index = 0;
+
+    public function __get($parameterName)
+    {
+        if ('index' === $parameterName) {
+            return $this->getHistoryIndex();
+        }
```

Comment:
We need to handle other arguments, too - see `TModelBase::__get` for an example on what to do.

Reply:
done